plat/mediatek/mt6795/include/mcucfg.h

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
MCU configuration register memory layout in 6795 is similar to 8173. However, in later SoC, the MCU configuration register differs, mcucfg.h content will change. mcucfg.h should still exist by platform SoC.